pages/profile.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Container, ProgressBar } from "react-bootstrap";
+
+export default function Profile() {
+  return (
+    <Container className="mt-5 text-center d-grid gap-2">
```

Comment:
I like how you used React Bootstrap here for the table. This will handle page responsiveness and resizing, so great choice to start simple! Besides, you can always add manual CSS later to style it